.devcontainer/docker-compose-local.yaml

```diff
@@ -20,7 +20,7 @@ services:
     image: localstack/localstack:3.4.0
     platform: linux/amd64
     ports:
-      - 4566:4566
+      - "0.0.0.0:4566:4566"
```

Comment:
Can you add some comments on mapping this to localhost? (and the extra hosts section below)

Reply:
Sure! I actually removed this change as it was not required for localstack to work properly, but I added comments below